src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -1158,6 +1159,14 @@ async Task<string> IDurableOrchestrationClient.RestartAsync(string instanceId, b
                 throw new ArgumentException($"An orchestrastion with the instanceId {instanceId} was not found.");
             }
 
+            bool isInstaceNotCompleted = status.RuntimeStatus == OrchestrationRuntimeStatus.Running || status.RuntimeStatus == OrchestrationRuntimeStatus.Pending || status.RuntimeStatus == OrchestrationRuntimeStatus.Suspended;
```

Comment:
nit: Please wrap this into multiple lines so that it can be read without scrolling.